qa/suites/upgrade/jewel-x/ceph-deploy/jewel-luminous.yaml

```diff
@@ -18,6 +18,14 @@ overrides:
         osd sloppy crc: true
       client:
         rbd default features: 5
+openstack:
+- machine:
+    disk: 100
+- volumes:
+    count: 3
+    size: 30
+#  hard-coded machine type 
+machine_type: vps
```

Comment:
with this you will not need openstack: fragment,  we can remove this if we want to schedule either on vps or ovh ?

Reply:
Yes I know, want to keep for the time being, so we won't forget it.  It's harmless but never used, you're right